lib/public/views/Runs/ActiveColumns/runsActiveColumns.js

```diff
@@ -500,8 +500,8 @@ export const runsActiveColumns = {
     },
     aliceL3AndDipoleCurrent: {
         name: 'L3 / Dipole',
-        visible: false,
+        visible: true,
         filter: ({ aliceL3AndDipoleCurrentFilter }) => selectionDropdown(aliceL3AndDipoleCurrentFilter, { selectorPrefix: 'l3-dipole-current' }),
-        profiles: ['runsPerLhcPeriod', 'runsPerDataPass', 'runsPerSimulationPass'],
+        profiles: ['runsPerLhcPeriod', 'runsPerDataPass', 'runsPerSimulationPass', profiles.none, 'lhcFill', 'environment'],
```

Comment:
Why adding 'lhcFill' and 'environment' profiles?

Reply:
Only kept the profiles.none this time